src/goRunTestCodelens.ts

```diff
@@ -69,7 +68,12 @@ export class GoRunTestCodeLensProvider implements CodeLensProvider {
 					arguments: [ { functionName: func.name} ]
 				};
 
-				let config = Object.assign({}, this.debugConfig, { args: ['-test.run', func.name], program: path.dirname(document.fileName) });
+				const args = ['-test.run', func.name];
+				const program = path.dirname(document.fileName);
+				const env = vsConfig['testEnvVars'] || {};
```

Comment:
From the codelens point of view, It's just creating a VSCode debug configuration with `env` and `envFile` that is later processed/merged by GoDelve, like it does with any user-configured `launch.json`.